non_linear/graphs/bellmann_ford/BellmanFord.py

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
remove the empty line on top.

Reply:
I usually follw this style ,
sorry for that :xd